README.md

```diff
@@ -662,6 +662,14 @@ Avoid ambiguity.
 
 **Versions X** and NOT **VX** or **version X**.
 
+## Terminology
+
+### Bus
+
+We agreed that we the word `Bus` should be avoided. Some replacements incldue:
+ * When referring to the topology, use decentralized or federated (as opposed to centralized e.g. RabbitMQ)
```

Comment:
Here's the page that needs to be updated: http://docs.particular.net/nservicebus/transports/. I'll do it.